File: JS-Basics/06-Conditional-Statements-Advanced-Exercise/p07_Hotel_Room.js
function hotelRoom(input) {
    let mes = input[0];
    let days = Number(input[1]);
    let apartmentPrice = 0;
    let studioPrice = 0;

    if (mes == 'May' || mes == 'October') {
        studioPrice = 50;
        apartmentPrice = 65;
        if (days > 14) {
            studioPrice -= studioPrice * 0.30;
        } else if (days > 7) {
            studioPrice -= studioPrice * 0.05;
        }
    } else if (mes == 'June' || mes == 'September') {
        studioPrice = 75.20;
        apartmentPrice = 68.70;
        if (days > 14) {
            studioPrice -= studioPrice * 0.20;
        }
    } else if (mes == 'July' || mes == 'August') {
        studioPrice = 76;
        apartmentPrice = 77;
    }

    if (days > 14) {
        apartmentPrice -= apartmentPrice * 0.10;
    }

    let totalPriceStudio = (studioPrice * days).toFixed(2);
    let totalPriceApartment = (apartmentPrice * days).toFixed(2);

    console.log(`Apartment: ${totalPriceApartment} lv.`);
    console.log(`Studio: ${totalPriceStudio} lv.`);

}

hotelRoom(["May", "15"]);
hotelRoom(["June", "14"]);

File: Programming-Basic/06-Conditional-Statements-Advanced-Exercise/p07_Hotel_Room.java
import java.util.Scanner;

public class p07_Hotel_Room {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String month = scanner.nextLine();
        int numOfOvernight = Integer.parseInt(scanner.nextLine());

        double pricePerNightsAp = 0.0;
        double pricePerNightsStudio = 0.0;
        double totalSumAp = 0.0;
        double totalSumStudio = 0.0;

        if (month.equals("May") || (month.equals("October"))) {
            pricePerNightsAp = 65;
            pricePerNightsStudio = 50;
            totalSumAp = numOfOvernight * pricePerNightsAp;
            if (numOfOvernight <= 7) {
                totalSumStudio = numOfOvernight * pricePerNightsStudio;
            } else if (numOfOvernight > 7 && numOfOvernight <= 14) {
                totalSumStudio = (numOfOvernight * pricePerNightsStudio) * 0.95;
            } else if (numOfOvernight > 14) {
                totalSumStudio = (numOfOvernight * pricePerNightsStudio) * 0.70;
            }
        }
        if (month.equals("June") || (month.equals("September"))) {
            pricePerNightsAp = 68.70;
            pricePerNightsStudio = 75.20;
            totalSumAp = numOfOvernight * pricePerNightsAp;
            if (numOfOvernight <= 14) {
                totalSumStudio = numOfOvernight * pricePerNightsStudio;
            } else {
                totalSumStudio = (numOfOvernight * pricePerNightsStudio) * 0.80;
            }

        }
        if (month.equals("July") || (month.equals("August"))) {
            pricePerNightsAp = 77;
            pricePerNightsStudio = 76;
            totalSumAp = numOfOvernight * pricePerNightsAp;
            totalSumStudio = numOfOvernight * pricePerNightsStudio;
        }

        if (numOfOvernight > 14) {
            totalSumAp = (numOfOvernight * pricePerNightsAp) * 0.90;

        }
        System.out.printf("Apartment: %.2f lv.%n", totalSumAp);
        System.out.printf("Studio: %.2f lv.", totalSumStudio);
    }
}
